go.mod

```diff
@@ -66,18 +66,18 @@ require (
 	gopkg.in/go-playground/webhooks.v5 v5.11.0
 	gopkg.in/src-d/go-git.v4 v4.13.1
 	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v1.17.8
-	k8s.io/apiextensions-apiserver v1.17.8
-	k8s.io/apimachinery v1.17.8
+	k8s.io/api v1.18.6
+	k8s.io/apiextensions-apiserver v1.18.6
+	k8s.io/apimachinery v1.18.6
```

Comment:
@alexmt As I have discovered the hard way, these should be `v0.18.6`, not `v1.18.6`. The only exception is `k8s.io/kubernetes` where the former is the right version and the later is not.

Reply:
👍 thank you! fixed.